x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -424,3 +425,68 @@ SETTING_WS
     : WS -> channel(HIDDEN)
     ;
 
+
+//
+// METRICS command
+//
+mode METRICS_MODE;
+METRICS_PIPE : PIPE -> type(PIPE), popMode;
+
+fragment METRICS_UNQUOTED_IDENTIFIER_PART
+    : ~[=`|,[\]/ \t\r\n]
+    | '/' ~[*/] // allow single / but not followed by another / or * which would start a comment
+    ;
+
+METRICS_UNQUOTED_IDENTIFIER
+    : METRICS_UNQUOTED_IDENTIFIER_PART+ -> popMode, pushMode(CLOSING_METRICS_MODE)
+    ;
+
+METRICS_QUOTED_IDENTIFIER
+    : QUOTED_IDENTIFIER -> type(QUOTED_IDENTIFIER), popMode, pushMode(CLOSING_METRICS_MODE)
+    ;
```

Comment:
Please remove this for now, as we might opt for a different syntax/parsing for quoted locations.